routes/userRouter.js

```diff
@@ -2,10 +2,10 @@ const express = require('express')
 const { signUp, login } = require('../controller/usercontroller')
 const userRouter = express.Router()
 
-userRouter.post('/signup',signUp)
-userRouter.post('/login',login)
+userRouter.post('/signup', signUp)
+userRouter.post('/login', login)
 
-userRouter.post('/login',(req,resp)=>{
+userRouter.post('/login', (req, resp) => {
     resp.send("login")
 })
```

Comment:
Why two routes with same endpoint are defined?

Reply:
sorry , i forgget to remove one